misc.go

```diff
@@ -112,6 +121,14 @@ func postWithMultipartResponse(ctx context.Context, path, name, fieldname string
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode == 429 {
+		retry, err := strconv.ParseUint(resp.Header.Get("Retry-After"), 10, 64)
+		if err != nil {
+			retry = 1
```

Comment:
I doubt 1 sec is going to be enough. When Slack rate limits they set the `Retry-After` to 60 seconds so if we ever get here I suggest we return 60 (which is crazily aggressive, I know, but not sure what other value would assure waiting enough time?).